Phonebook.Frontend/src/app/pages/page-information/page-information.component.scss

```diff
@@ -1,3 +1,25 @@
-.small-card {
-  width: 50%;
+.mat-card{
+  display: flex;
+  flex-direction: column;
+  flex: 1;
 }
+
+// mat-card-footer doesn't align with bottom, you have to user manually styling
+// --> https://github.com/angular/components/issues/11094
+.mat-card-content{
+  flex-grow: 1;
+  overflow: auto;
+}
+
+.mat-card-action {
+  margin-bottom: 0;
+}
+// manually styling ends here
+
+@media (max-width: 767px) {
```

Comment:
You can use media-queries in `media-queries.scss` for this

Reply:
Nope, that's site specific!

Reply:
Uhh, nvm. Of course please use the predefined break points :)